src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs

```diff
@@ -310,7 +315,7 @@ public Task TestInProcessFeaturesNotEnabled(Architecture targetArchitecture)
                 });
         }
 
-        [Theory(Skip = "Disable unstable tests.")]
+        [Theory]
```

Comment:
I just put out a PR that does this check for all tests that use `ScenarioHelper.SingleTarget`, which includes this test: #2990

Reply:
Since this is a trigger-based test that suspends the application, I'm not sure this is the issue, but I will do some testing around it. I will disable the test for this PR.